src/merge_status_label.py

```diff
@@ -6,7 +7,7 @@ def merge_status_label(pull, commit):
         print("No commit given")
         return
 
-    label = "Ready for merge"
+    label = READY_FOR_MERGE
```

Comment:
Can be removed and used directly `label = READY_FOR_MERGE`

Reply:
done